cmd/kube-proxy/app/server_others.go

```diff
@@ -183,25 +181,28 @@ func newProxyServer(
 	iptInterface = utiliptables.New(execer, primaryProtocol)
 
 	var ipt [2]utiliptables.Interface
-	dualStack := utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) && proxyMode != proxyModeUserspace
-	if dualStack {
+	dualStack := true // While we assume that node supports, we do further checks below
+
+	if proxyMode != proxyModeUserspace {
 		// Create iptables handlers for both families, one is already created
 		// Always ordered as IPv4, IPv6
 		if primaryProtocol == utiliptables.ProtocolIPv4 {
 			ipt[0] = iptInterface
 			ipt[1] = utiliptables.New(execer, utiliptables.ProtocolIPv6)
-
-			// Just because the feature gate is enabled doesn't mean the node
-			// actually supports dual-stack
-			if _, err := ipt[1].ChainExists(utiliptables.TableNAT, utiliptables.ChainPostrouting); err != nil {
-				klog.Warningf("No iptables support for IPv6: %v", err)
-				dualStack = false
-			}
 		} else {
 			ipt[0] = utiliptables.New(execer, utiliptables.ProtocolIPv4)
 			ipt[1] = iptInterface
 		}
+
+		// TODO: @khenidak - should we fail here? Also no checks for single-stack IPv6 clusters.
+		// MUST BE ADDRESSED PART OF KUBE-PROXY DUAL STACK STABLE
+		// 1: is always IPv6 ipt
+		if _, err := ipt[1].ChainExists(utiliptables.TableNAT, utiliptables.ChainPostrouting); err != nil {
```

Comment:
Should we just add `ipt.Present() bool` to encapsulate this "does it work" check?